examples/rosettacode/arithmetic_mean.nit

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: Arithmetic mean
+# SEE: <http://rosettacode.org/wiki/Averages/Arithmetic_mean>
+module arithmetic_mean
+
+# Computes the meam or the `array` values.
+fun mean(array: Array[Int]): Float do
```

Comment:
I thing you should pass an Array[Float] to simplify the code and be more type consistant